### PR TITLE
Add taranaki well logs dataset schema

### PR DESCRIPTION
**Pull request by djalova**

### CODE REVIEW

**Comment by xuhdev** [comment]
Turns out this is still blocked... Let's prioritize issues that are blocking these changes

**Comment by edwardleardi** [comment]
@xuhdev Which issue is blocking this PR?

**Comment by xuhdev** [comment]
@edwardleardi https://github.com/codait/pydax/issues/50 . You should be able to see if you have installed ZenHub browser extension

File: datasets.yaml [hunk 1]
```diff
@@ -151,3 +151,56 @@ datasets:
...
+                WELLNAME: 'string'
+                FORMATION: 'string'
+          path: taranaki-basin-curated-well-logs/logs.csv
+        coordinates:
+          name: Taranaki Basin Curated Well Coordinates
+          description: Coordinates of the wells
+          format:
+            id: csv
+            options:
+              WELL_NAME: 'string'
```
Comment (xuhdev) [inline datasets.yaml:201]: You forgot `columns`

**Review by xuhdev — CHANGES_REQUESTED** [review]
Missing `columns:`

**Review by xuhdev — APPROVED** [review]
LGTM

**Pull request merged**